dev-java/commons-junit4/commons-junit4-1.2.17.ebuild

```
@@ -0,0 +1,39 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+JAVA_PKG_IUSE="doc source test"
+MAVEN_ID="de.unkrig.commons:commons-junit4:1.2.17"
+JAVA_TESTING_FRAMEWORKS="junit-4"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="de.unkrig.commons - A general-purpose Java class library"
```

```suggestion
DESCRIPTION="Assertions etc. for those who write tests with JUNIT 4"
```

source: https://unkrig.de/w/Commons.unkrig.de

sure could be improved, but better as what we have right now